apps/design-land/src/app/loading-icon/loading-icon.component.html

```diff
@@ -1,2 +1,33 @@
-<daff-loading-icon [diameter]="50" color="secondary"></daff-loading-icon>
-<daff-loading-icon [diameter]="50"></daff-loading-icon>
\ No newline at end of file
+<daff-article>
+  <h1 daffArticleTitle>Loading Icon</h1>
+  <p daffArticleLead>
+    The loading icon is used as an indicator of an event in progress.
+  </p>
+
+  <h2>About</h2>
+  <p>
+    This loading icon can be used to indicate that an event is ocurring and is still in progress.
+    The color and size of the loading icon can be customized to suit the needs of your project.
+  </p>
+
+  <h3>Diameter</h3>
+  <design-land-example-viewer example="loading-icon-diameter"></design-land-example-viewer>
+
+  <h3>Theming</h3>
+  <p>
+    The loading icon color can be changed by using the <code>color</code> property. The default color is set to
+    <code>theme-contrast</code>.
+    This can be changed to one of the supported colors.
+  </p>
+  <p>Supported colors: <code>primary | secondary | tertiary | black | white | theme | theme-contrast</code></p>
+  <design-land-example-viewer example="loading-icon-color"></design-land-example-viewer>
+
+  <h3>Accessibility</h3>
+  <p>
+    You should ensure that you use a meaningful label with <code>aria-label</code> or <code>aria-labelledby</code>.
```

Comment:
Loading icons should be given meaningful labels by using <code>aria-label</code> or <code>aria-labelledby</code>. Additionally, if a loading icon is used to indicate a process in progress, using `aria-live` and `aria-busy` should be strongly considered.

Reply:
bbc47aaee1e8688e5fe006ad6c0e1ef8ff412fed